elephant/signal_processing.py

```diff
@@ -611,3 +611,160 @@ def hilbert(signal, N='nextpow'):
     output = signal.duplicate_with_new_data(
         scipy.signal.hilbert(signal.magnitude, N=n, axis=0)[:n_org])
     return output / output.units
+
+
+def rauc(signal, baseline=None, bin_duration=None, t_start=None, t_stop=None):
+    '''
+    Calculate the rectified area under the curve (RAUC) for an AnalogSignal.
+
+    The signal is optionally divided into bins with duration `bin_duration`,
+    and the rectified signal (absolute value) is integrated within each bin to
+    find the area under the curve. The mean or median of the signal or an
+    arbitrary baseline may optionally be subtracted before rectification. If
+    the number of bins is 1 (default), a single value is returned for each
+    channel in the input signal. Otherwise, an AnalogSignal containing the
+    values for each bin is returned along with the times of the centers of the
+    bins.
+
+    Parameters
+    ----------
+    signal : neo.AnalogSignal
+        The signal to integrate. If `signal` contains more than one channel,
+        each is integrated separately.
+    bin_duation : quantities.Quantity
```

Comment:
```suggestion
    bin_duration : quantities.Quantity
```

Reply:
Good catch! I found a second place where this typo was present and fixed it as well.